client/my-sites/themes/controller.js

```diff
@@ -68,6 +69,19 @@ function makeElement( ThemesComponent, Head, store, props ) {
 	);
 };
 
+// Where do we put this? It's client/server-agnostic, so not in client/controller,
+// which requires ReactDom... Maybe in lib/react-helpers?
```

Comment:
I just wanted to separate concerns somewhat, and `client/controller` is currently concerned with client-only -- so I didn't want to dilute that. (As for naming, that file could be moved to something like `app/client-helpers/` later, which e.g. could also contain `side-effects`.)